core/src/main/resources/hudson/Messages_fr.properties

```
@@ -49,16 +47,82 @@ Util.day   ={0} j
+ Jenkins refusera de charger les dépendances nécessitant une version plus récente de Jenkins, \
+ et en conséquence ce plugin échouera à se charger.
+PluginManager.securityWarning=\
+  Avertissement : cette version de plugin n''est peut-être par sûre. Veuillez vérifier les notices de sécurité suivantes :
+PluginManager.ago=Il y a {0}
+PluginManager.adoptThisPlugin=\
+  <strong>Ce plugin est à adopter!</strong> Nous recherchons de nouveaux mainteneurs. \
+  Veuillez consulter notre initiative <a href="https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/" rel="noopener noreferrer" target="_blank">Adopter un plugin</a> pour plus d''informations.
+PluginManager.deprecationWarning=<strong>Ce plugin est obsolète.</strong> Généralement, cela signifie qu''il est soit obsolète, plus développé, ou ne fonctionne peut-être plus. <a href="{0}" rel="noopener noreferrer" target="_blank">En apprendre plus.</a>
+PluginManager.insecureUrl=\
+  Vous utilisez une URL non sécurisée pour télécharger le plugin, utilisez-la à vos propres risques et périls!
+PluginManager.invalidUrl=\
+  Vous utilisez une URL invalide pour télécharger le plugin. Les URLs file, https et http (non recommandé) sont supportées.
+
+PluginManager.emptyUpdateSiteUrl=\
+  Le site de mise à jour ne peut être vide. Veuillez entrer une URL valide.
+
+PluginManager.connectionFailed=\
+  Impossible de se connecter à l''URL.
+
+AboutJenkins.DisplayName=À propos de Jenkins
 AboutJenkins.Description=Afficher les informations de version et de licence
 
 ProxyConfiguration.TestUrlRequired=Une URL de test est requise.
+ProxyConfiguration.MalformedTestUrl=L''URL de test {0} n''est pas correctement formée.
 ProxyConfiguration.FailedToConnectViaProxy=Impossible de se connecter à {0}.
 ProxyConfiguration.FailedToConnect=Impossible de se connecter à {0} (code {1}).
 ProxyConfiguration.Success=Succès
+
+Functions.NoExceptionDetails=Aucun détail concernant l''exception
+
+PluginWrapper.missing=Il manque le plugin : {0} ({1})
+PluginWrapper.failed_to_load_plugin_2=Échec du chargement de : {0} ({1} {2})
+PluginWrapper.failed_to_load_dependency_2=Échec du chargement de : {0} ({1} {2})
+PluginWrapper.disabled_2=Le plugin nécessaire suivant est désactivé : {0} ({1})
+PluginWrapper.obsolete_2=Mise à jour nécessaire : {0} ({1} {2}) doit être mis à jour en {3} ou plus
+PluginWrapper.obsoleteCore=Jenkins ({1}) ou plus récent nécessaire
+PluginWrapper.PluginWrapperAdministrativeMonitor.DisplayName=Plugins échouant à charger
+PluginWrapper.Already.Disabled=Le plugin ''{0}'' était déjà désactivé
+PluginWrapper.Plugin.Has.Dependent=Le plugin ''{0}'' a au moins un plugin dépendant ({1}) et la stratégie de désactivation est {2} ainsi, il ne peut pas être désactivé
+PluginWrapper.Plugin.Disabled=Plugin ''{0}'' désactivé
+PluginWrapper.NoSuchPlugin=Aucun plugin trouvé avec le nom ''{0}''
+PluginWrapper.Error.Disabling=Une erreur a été relevée lors de la désactivation du plugin ''{0}''. Erreur : ''{1}''
```

```suggestion
PluginWrapper.Already.Disabled=Le plugin "{0}" était déjà désactivé
PluginWrapper.Plugin.Has.Dependent=Le plugin "{0}" a au moins un plugin dépendant ({1}) et la stratégie de désactivation est {2} ainsi, il ne peut pas être désactivé
PluginWrapper.Plugin.Disabled=Plugin "{0}" désactivé
PluginWrapper.NoSuchPlugin=Aucun plugin trouvé avec le nom "{0}"
PluginWrapper.Error.Disabling=Une erreur a été relevée lors de la désactivation du plugin "{0}". Erreur : "{1}"
```
Use of the double quotes character instead of doubling the single quote character.